content/tooling/wrangler/webpack.md

```diff
@@ -94,6 +94,31 @@ module.exports = {
 }
 ```
 
+## Shimming globals
+
+Sometimes you want to bring your own implementation of an existing global API. You can do this by [shimming](https://webpack.js.org/guides/shimming/#shimming-globals) a third party module in its place as a webpack plugin.
+
+For example, to replace the runtime global `URL` class with the npm package `url-polyfill` - or your choice of third party package - `npm i` the package and add a plugin entry to your webpack config.
```

Comment:
does this need to be a global npm install or can it just be a local install for this worker? Assuming the latter, but it would be good to be explicit about that.

Reply:
i could also just say "add it to your package.json" and that would probably be clearer.

Reply:
How about
```suggestion
For example, to replace the runtime global `URL` class with the npm package `url-polyfill` - or your choice of third party package - `npm i` the package to install it locally and add it to your worker's package.json, then add a plugin entry to your webpack config.
```